modules/nf-core/hifiasm/hifiasm.diff

```diff
@@ -13,7 +13,7 @@ Changes in module 'nf-core/hifiasm'
 +        exit 1, "This version of HIFIASM module does not support Conda. Please use Docker / Singularity / Podman instead."
 +    }
 +
-+    container "wave.seqera.io/wt/73ac3caec075/wave/build:hifiasm-0.19.8_samtools-1.20--1f6824530f0d0ad5"
++    container "quay.io/sanger-tol/hifiasm_samtools:0.01"
```

Comment:
This docker image is still manually built and no version control. 
Any way you can push this path to nf-core now?

Reply:
We keep it as is for now, as we don't push the patch into nf-core.